### PR TITLE
Remove all uCFG-related classes and files

### PR DESCRIPTION
**Pull request by valhristov**

### CODE REVIEW

File: sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/AspNetMvcHelper.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 namespace SonarAnalyzer.ControlFlowGraph.CSharp
 {
-    public static class TaintAnalysisEntryPointDetector
+    public static class AspNetMvcHelper
```
Comment (Evangelink) [inline sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/AspNetMvcHelper.cs]: Shouldn't belong to this PR.

File: sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/MethodShouldBeNameAccordingToSynchronicity.cs [hunk 2]
```diff
@@ -65,7 +66,8 @@ protected override void Initialize(SonarAnalysisContext context)
                         methodSymbol.IsMainMethod() ||
                         methodSymbol.GetInterfaceMember() != null ||
                         methodSymbol.GetOverriddenMember() != null ||
-                        methodSymbol.IsTestMethod())
+                        methodSymbol.IsTestMethod() ||
```
Comment (Evangelink) [inline sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/MethodShouldBeNameAccordingToSynchronicity.cs]: Same here

File: sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/FrameworkMetadataReference.cs [hunk 3]
```diff
@@ -41,6 +41,9 @@ internal static class FrameworkMetadataReference
         internal static MetadataReference Mscorlib { get; }
             = Create("mscorlib.dll");
 
+        internal static MetadataReference Netstandard { get; }
```
Comment (Evangelink) [inline sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/FrameworkMetadataReference.cs]: Should not be part of this PR.

File: sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Helpers/AspNetMvcHelper_IsControllerMethod.cs [hunk 4]
```diff
@@ -26,7 +26,7 @@
 namespace SonarAnalyzer.UnitTest.ControlFlowGraph
 {
     [TestClass]
-    public class EntryPointRecognizer_IsEntryPoint
+    public class AspNetMvcHelper_IsControllerMethod
```
Comment (Evangelink) [inline sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Helpers/AspNetMvcHelper_IsControllerMethod.cs]: Should not be part of this PR.

File: sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/MethodShouldBeNameAccordingToSynchronicityTest.cs [hunk 5]
```diff
@@ -40,6 +40,34 @@ public void MethodShouldBeNameAccordingToSynchronicity(string tasksVersion)
                 additionalReferences: NuGetMetadataReference.SystemThreadingTasksExtensions(tasksVersion));
         }
 
+        [TestMethod]
```
Comment (Evangelink) [inline sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/MethodShouldBeNameAccordingToSynchronicityTest.cs]: Should not be part of this PR.

File: sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MethodShouldBeNameAccordingToSynchronicity.MVC.Core.cs [hunk 6]
```diff
@@ -0,0 +1,46 @@
+﻿using System;
```
Comment (Evangelink) [inline sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MethodShouldBeNameAccordingToSynchronicity.MVC.Core.cs]: Should not be part of this PR.

File: sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MethodShouldBeNameAccordingToSynchronicity.MVC.cs [hunk 7]
```diff
@@ -0,0 +1,27 @@
+﻿using System;
```
Comment (Evangelink) [inline sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MethodShouldBeNameAccordingToSynchronicity.MVC.cs]: Should not be part of this PR.

**Comment by Evangelink** [comment]
For some reason this PR contains 2 commits (the first one is totally unrelated to this PR).

File: sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/AspNetMvcHelper.cs (hunk 1 above)
Comment (valhristov) [reply sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/AspNetMvcHelper.cs]: Hence the [DO NOT MERGE]
